components/tensorflow-notebook-image/Dockerfile

```diff
@@ -123,8 +123,16 @@ RUN conda create -n py2 python=2 && \
     && \
     python -m ipykernel install && \
     # tensorflow-model-analysis is only supported for TF 1.6 and above
-    if [[ $INSTALL_TFMA == "yes" ]]; then \
-      pip install --no-cache-dir tensorflow-model-analysis && \
+    if [[ $TFMA_VERSION != "disable_install" ]]; then \
```

Comment:
How about just 'none'?

Reply:
I think 'disable_install' is clear, somebody may think the none as any version. I think 'disable_install' is a more elegant code style.

Reply:
I think using magic values makes this unnecessarily complicated.

We'd like builds to be repeatable. So pinning python packages to specific versions is the best approach.

So lets check if the value is specified and if its is that should be the version of the package to install.

We can use the empty string to indicate don't install this.

I'd really like to get this into the 0.3.1 release so lets make this changes and if we want to discuss whether that's the right solution lets do it in a follow on issue.